### PR TITLE
[CALC] de-DE: add menu-accels, *.rc: Portu, CTRL->Ctrl, STRG->Strg

### PR DESCRIPTION
**Pull request by JoachimHenze**
- de-DE: add accelerators for the menus, inspired by german XPSP3
- *.rc: It is portuguese, not portoguese (this has been fixed in en-US earlier already, but was forgotten to be merged into other languages
- *.rc: CTRL is not used in capital manner by MS, but MS used Ctrl, and MS did not use STRG but Strg for de-DE. Our en-US respected that already, but again stuff was not synced into other languages when it was improved earlier in ros

Therefore this PR has absolutely no changes for en-US because that was good already.

JIRA issue: none

### CODE REVIEW

File: base/applications/calc/lang/de-DE.rc [hunk 1]
```diff
@@ -307,89 +307,89 @@ END
...
-        MENUITEM "Grad\tF4", IDM_VIEW_GRAD, CHECKED
+        MENUITEM "&Deg\tF2", IDM_VIEW_DEG, CHECKED
+        MENUITEM "&Rad\tF3", IDM_VIEW_RAD, CHECKED
+        MENUITEM "&Grad\tF4", IDM_VIEW_GRAD, CHECKED
         MENUITEM SEPARATOR
-        MENUITEM "Zifferngruppierung", IDM_VIEW_GROUP, CHECKED
+        MENUITEM "Zifferngr&uppierung", IDM_VIEW_GROUP, CHECKED
     END
-    POPUP "Hilfe"
+    POPUP "&?"
```
Comment (HBelusca) [inline base/applications/calc/lang/de-DE.rc:332]: Why the change?

Comment (JoachimHenze) [reply base/applications/calc/lang/de-DE.rc:332]: Because that is what MS Calc uses here (pic from german MS XPSP3 calc attached)
image

**Review by HBelusca — APPROVED** [review]
